### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by berk76** [underground-lab/eu2020#22] [Ideas]
Ahoj Michale,

ve Wiki jsem vytvořil stránku s postupem jak udělat release. Vyzkoušíš?

Jarda

**Comment by mportesdev** [comment]
Ano,
postup znám, udělám.

U jednoho projektu používám automatické vytvoření release v GitHub Actions:

https://github.com/softprops/action-gh-release

https://github.com/mportesdev/handpick/blob/main/.github/workflows/build-publish-release.yml#L32

(ale jen jednoduchý, s nadpisem a bez textu v těle)

**Comment by berk76** [comment]
Klidně to zkusme.
